sklearn/tests/test_random_projection.py

```diff
@@ -377,6 +378,51 @@ def test_random_projection_feature_names_out(random_projection_cls):
     assert_array_equal(names_out, expected_names_out)
 
 
+@pytest.mark.parametrize("random_state", range(50))
+@pytest.mark.parametrize("n_rows", (2, 9, 10, 11, 1000))
+@pytest.mark.parametrize("n_cols", (2, 9, 10, 11, 1000))
+@pytest.mark.parametrize("random_projection_cls", all_RandomProjection)
```

Comment:
Wow that makes a lot of tests ! 2500, right ?

We usually don't parametrize tests by seed. It would make the test suite take way too much time. We're thinking about setting up a mechanism to test for different seeds from time to time, but it's another matter.

In the mean time let's just set random_state=0.

Reply:
I added this because one test actually passed although there was a bug for some random seeds. IIRC, it was because the random matrix had some empty rows in some cases. But okay, I'll set `random_state=0`.

Reply:
A agree it's a good thing to test against many seeds to check that it's not passing by luck. We are setting up a way to run the tests with a different seed each time in https://github.com/scikit-learn/scikit-learn/pull/22749.